lib/pages/home.dart

```diff
@@ -2,35 +2,69 @@ import 'package:flutter/widgets.dart';
 
 import '../widgets/button.dart';
 import '../widgets/scaffold.dart';
+import '../widgets/bottom_navigate.dart';
 
-class Home extends StatelessWidget {
+import './select_pixel.dart';
+import './feed/pixel_list.dart';
+
+class Home extends StatefulWidget {
+  Home();
   @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-        body: SafeArea(
-      child: Center(
-        child: Column(
-          children: [
-            Text('My Pixels'),
-            Button(
-              label: 'See pixel',
+  _HomeState createState() => _HomeState();
+}
+
+class _HomeState extends State<Home> {
+  static BuildContext _context;
+  static int _currentIndex;
+
+  final List<Widget> _pages = [
+    PixelList(),
+    Text('search'),
+    SelectPixel(),
+    Center(
```

Comment:
Move this code to another widget to keep code clean

Reply:
:+1:
Done